client/lib/url/index.js

```diff
@@ -24,14 +26,28 @@ function isOutsideCalypso( url ) {
 }
 
 function isExternal( url ) {
-	const { hostname } = parseUrl( url, false, true ); // no qs needed, and slashesDenoteHost to handle protocol-relative URLs
+	// parseURL will return hostname = null if no protocol or double-slashes
+	// the url passed in might be of form `en.support.wordpress.com`
+	// so for this function we'll append double-slashes to fake it
+	// if it is a relative URL the hostname will still be empty from parseURL
+	if ( ! includes( url, '//' ) ) {
```

Comment:
This condition wouldn't catch the following URL: `wordpress.com//themes` (which although a typo, still works)